src/FsAutoComplete/FsAutoComplete.fsproj

```diff
@@ -2,12 +2,15 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net461</TargetFramework>
+    <TargetFrameworks>net461;netcoreapp2.0</TargetFrameworks>
```

Comment:
Is .NET Framework needed as a TFM still?

Reply:
I'd like to keep it because it includes the old ProjectCrackerVerbose, which is a wrapper around the .net-framework only project cracker, which may be better-supported/more-compatible with old-style project files.  If/when we drop it, though, we could definitely standalone-publish FSAC and have a xcopy-deployable standalone version which would be pretty neat.

Reply:
>Is .NET Framework needed as a TFM still?

As far as I know, Ionide is only editor using Core version of the FSAC.

> which may be better-supported/more-compatible with old-style project files

I definitely had some issues where new project parser fails for an odd old-style project, while FCS' ProjectCrapper works just fine... I think it's even happening on the VF# repo.

Reply:
Main issue as @Krzysztof-Cieslak said is the different project loader atm.
plus vim/emacs only use .net version (but that's easy to change in these extensions)

.NET is needed because there the `projectcracker` is used to load the old fsproj (verbose fsproj) loading the msbuild lib too.

With the `dotnet-proj-info` project parser, usually the fsproj loading fails if the msbuild on PATH is wrong (usually wrong visual studio command prompt). i was able to load the VF# sln in ionide.

I am addressing that in `dotnet-proj-info` repo, i hope to have a new release soon, refacotring out the project loading from FSAC and embedding the discovery and choice of msbuild to use in `dotnet-proj-info`.

We cannot use the new `dotnet-proj` on .net FSASC in parallel (with a feature flag), because it will conflict with the msbuild libs loaded by projectcracker